### DISCUSSION TITLE
The bard_html modifier was not found.

### DISCUSSION THREAD

**Discussion by tcagkansokmen** [statamic/cms#8755] [Q&A]
Hi there,

I've installed "cookie byte" add-on. And it has line like below;

`{{ modal_description | raw | bard_html }}`

But, It returns error like that.
I tried to find how to install this modifier, but its default modifier, so it must be added automatically. But it doesnt exist. Whats wrong with my build? How can I add "bard_html" to my build?

<img width="1225" alt="image" src="https://github.com/statamic/cms/assets/3479434/35820e94-2601-46c8-a3db-1ea00f7feca7">

**Comment by jasonvarga** [comment]
That modifier was added in 3.3.29. Make sure you are at least on that version.
